src/lib/components/consignments/components/artwork_consignment_todo.tsx

```diff
@@ -86,45 +86,58 @@ const DoneButton = () =>
     <Image source={require("../images/black-tick.png")} />
   </ImageBG>
 
+const Button = (props: TouchableHighlightProperties) =>
```

Comment:
Interesting, this does look like it should do it.

I _think_ this is more of a case that touchable highlight doesn't expect children (it can only have one child, which can have more)  as adding the specifier didn't help

![screen shot 2017-06-14 at 13 36 29](https://user-images.githubusercontent.com/49038/27132384-9da5d916-5106-11e7-9d8c-fe803b87efc5.png)